start_cmd_qldb.bat

```diff
@@ -0,0 +1,3 @@
+echo off
+REM Initiate a cmd, go to user's home path and run qldb.exe help
+cmd /Q /K "cd %HOMEPATH% && qldb.exe -h"
```

Comment:
Does the -h need to be here, or can we launch directly into the shell?

Reply:
The -h means show the help.

Reply:
The qldb shell needs a ledger to start it properly; it won't launch into a shell without it.

Reply:
Ack, that sucks.